cmake/modules/FindMaya.cmake

```diff
@@ -169,19 +168,6 @@ find_path(MAYA_INCLUDE_DIR
         "Maya's headers path"
 )
 
-find_path(MAYA_LIBRARY_DIR
```

Comment:
This is not needed. It is duplicated from above in each platform specific section.